Sources/core/objects/Human.cpp

```diff
@@ -514,6 +556,43 @@ void Human::Bump(IdPtr<IMovable> item)
     IMovable::Bump(item);
 }
 
+void Human::RotationAction(IdPtr<IOnMapBase> item)
+{
+    if (IdPtr<IMovable> movable = item)
+    {
+        if (!movable->anchored)
+        {
+            if (IdPtr<Projectile> projectile = movable)
+            {
+                return;
+            }
+            movable->Rotate((movable->GetDir() + 1) % 4);
+        }
+    }
+}
+
+void Human::PullAction(IdPtr<IOnMapBase> item)
+{
+    if (IdPtr<IMovable> movable = item)
+    {
+        if (CanTouch(movable))
+        {
+            if (IdPtr<Projectile> projectile = movable)
+            {
+                return;
+            }
+            pulled_object_ = movable;
+            interface_.UpdatePulling(true);
+        }
+    }
+}
+
+void Human::SetPullToNull()
```

Comment:
Better just name it `StopPull`. Current name describes internal behaviour, and it is not really needed.